frontend/src/views/Combat.vue

```diff
@@ -535,7 +649,7 @@ export default {
       }
       return x1 + x2;
     },
```

Comment:
We could remove this crazy regex addCommas() and just change L129 to `{{e.power.toLocaleString()}}`, where it is used.